agent-side-integrations/crm-helpdesk-integrations/dynamics.md

```diff
@@ -0,0 +1,131 @@
+  Cobrowse integrates seamlessly with Dynamics 365 Customer Service, enabling
+  real-time visual collaboration between agents and customers. This integration
+  is available through the Microsoft AppSource store.
+---
+
+# Dynamics 365 - Customer Service
+
+## Overview
+
+Cobrowse integrates seamlessly with Dynamics 365 Customer Service, enabling
+real-time visual collaboration between agents and customers. This integration is
+available through the
+[Microsoft AppSource store](https://appsource.microsoft.com/en-us/product/dynamics-365/cobrowseiollc.integration).
+
+{% embed url="https://vimeo.com/1095905912?share=copy" %} Cobrowse for Dynamics
+365 Customer Service {% endembed %}
+
+## Installation guide
+
+To install and configure the Cobrowse integration with Dynamics 365:
+
+1. Install the app from the
+   [Microsoft AppSource store](https://appsource.microsoft.com/en-us/product/dynamics-365/cobrowseiollc.integration).
+2. _(Optional)_ If you are
+   [self-hosting](../../../enterprise-self-hosting/self-hosting-overview.md)
+   your Cobrowse.io instance, follow the
+   [self-hosted instructions](#self-hosted).
+3. As a Dynamics administrator, open the Cobrowse.io application from the app
+   launcher. Under **Setup**, select the **OAuth Consent** link. This will
+   generate a URL that must be approved by an Azure administrator.
+
+### Enabling Cobrowse for live chat
+
+To enable Cobrowse functionality for live chat channels:
+
+1. Navigate to the **Copilot Service Admin Center** for your organization.
+2. Go to **Channels > Chat > Manage**.
+3. Edit your desired channel and open **User features**.
+4. Toggle the **Co-browse** feature on and select **Cobrowse.io** as the
+   provider.
+
+Note that to use the Cobrowse user feature with the Dynamics live chat widget,
+you must integrate the [Cobrowse SDK](../../sdk-installation/web.md) into the
+webpage where the widget is hosted.
+
+### Enabling Cobrowse for other channel types
+
+Cobrowse functionality is also available for other communication channels such
+as voice, WhatsApp, and more. While the agent will not automatically see the
+user’s device in these cases, a cobrowsing session can still be initiated using
+a [6-digit code](../../sdk-features/6-digit-codes.md).
+
+If your channel is already set up, follow these steps to enable Cobrowse:
+
+1. Navigate to the **Copilot Service Admin Center** for your organization.
+2. Go to **Workspaces > Session Templates > Manage**.
+3. Select the session template associated with your channel and click **Edit**.
+4. Under **Additional Tabs**, choose **Add Existing Application**
+5. Search for **Cobrowse.io Tab** and add it to the list
+6. Click **Save & Close**
+
+Note: Default session templates cannot be edited. To work around this, create a
+new session template, replicate the settings from the default template, and then
+add the Cobrowse.io application tab.
+
+This will add Cobrowse as a default tab when agents open an interaction.
```

Comment:
This section is all about supporting non-chat based support interactions (e.g. someone calling into the helpdesk to get support to troubleshoot their table arrangement app on a locked down ipad, but they have cobrowse installed in the app).

This is possible because we relaxed the auth constraint here:
https://github.com/cobrowseio/cobrowse-api/pull/195

It's kinda quirky though. The Cobrowse.io tab will be added immediately (but not loaded yet). So all the agent needs to do is click the tab, they'll be dropped in the sidebar with 0 devices listed, but they can use the 6-digit code field to start a session.